integration-test/tests/login-form.ios.spec.js

```diff
@@ -151,11 +151,37 @@ test.describe('Auto-fill a login form on iOS', () => {
                 await login.promptWasNotShown()
                 await login.assertDialogClose()
                 await login.openDialog()
-                await login.fieldsDoNotContainIcons()
+                await login.fieldsContainIcons()
 
                 await login.clickIntoUsernameInput()
                 await login.assertFormSubmitted()
             })
+            test('should prompt to store and not autosubmit when the form completes a partial credential stored', async ({page}) => {
```

Comment:
Several of these tests are exactly the same or with tiny variations across different platforms. I've created https://app.asana.com/0/1198964220583541/1204193691790851/f to circle back and find a better solution to write the tests once and run them on all platforms.